slides/js_ecosystem.md

```diff
@@ -505,23 +504,22 @@ You can use additional tools, provided they are installed with `yarn install`, u
   the necessary parts
 
 ---
-
 # Homework
 
 If you need to provide further information, include it in README.md.
 
-Submit zipped file to [https://courses.cs.ut.ee/2021/react/Main/Submit](https://courses.cs.ut.ee/2021/react/Main/Submit)
+Submit zipped file to [https://courses.cs.ut.ee/2022/react/Main/Submit](https://courses.cs.ut.ee/2022/react/Main/Submit)
 
 Don’t include node_modules or .git/, .hg/, .svn/, make sure that your zipped
 file size is reasonable.
 
 You can use/modify/extend [this script](https://gist.github.com/urmastalimaa/70edc0728cb711234f42) (usage: `node zipHomework.js`) to automate creating a zip file without specific folders.
 
 ---
+#Homework result
 
-# Homework result
+.full-image[![JS-tooling live reload](assets/lecture-1/hw-01-gif.gif)]
 
-<video width="100%" height="50%" controls>
-  <source src="assets/homework-mathemagician-1-result.m4v" type="video/mp4">
-  Your browser does not support the video tag.
-</video> 
+---
+class: middle, center
+#Questions?
```

Comment:
Minor note:
Generally with slides, "Questions?"/"Thanks for listening" end-slides are avoided. Here, similarly, it is best to "end" on the homework result GIF, which is going to be the most likely source of questions.